frameworks/PHP/cygnite-php-framework/apps/views/bench/fortunes.view.php

```diff
@@ -1,6 +1,9 @@
 <!DOCTYPE html>
 <html>
-<head><title>Fortunes</title></head>
+<head>
+    <title>Fortunes</title>
+</head>
+
```

Comment:
For future reference, we should all strive to not make changes outside of the PR's goal. I didn't see anything about `cygnite` in the original comment. Additionally, why add bytes when the purpose of the test is to show the performance of the platform? They add up over thousands of requests and are unnecessary.